packages/lesswrong/themes/globalStyles/globalStyles.ts

```diff
@@ -37,6 +37,10 @@ const clearStyle = (theme: ThemeType): JssStyles => ({
     border: "none",
     boxShadow: "none",
     cursor: "pointer",
+    background: "none",
+    '&:hover': {
+      opacity: 0.5,
+    }
```

Comment:
Doing this globally seems not-safe to me. Just checking the first button I see, it's definitely changed on hover:

Master:
![Screenshot 2022-11-01 at 11 43 46](https://user-images.githubusercontent.com/5075628/199225501-af164ed5-a860-484b-81f1-db43303c28d6.png)

This branch:
![Screenshot 2022-11-01 at 11 43 03](https://user-images.githubusercontent.com/5075628/199225532-24cb19f1-3293-4dd2-b9a5-21eb2cccedfd.png)

There's probably other places where `background: none` has made changes too.

Reply:
So on one hand, after some discussion I think we ended up deciding to use aria-role: "button" instead of using the <button> tag, so I may just retract this. But, insofar as we ever use button, the default button background looks pretty crappy and needs to be overridden. (AFAICT it's basically a mistake it wasn't already overridden in this global file)

But, can leave that for a different PR, and/or not worry about it since probably we won't use buttons that often.